src/cdk/table/coalesced-style-scheduler.ts

```diff
@@ -99,3 +145,10 @@ export class _CoalescedStyleScheduler implements OnDestroy {
       : this._ngZone.onStable.pipe(take(1));
   }
 }
+
+function flushTasks(tasks: (() => unknown)[]) {
```

Comment:
Can you add docs for this, especially since it mutates (empties) the given array